solver/src/driver.rs

```diff
@@ -134,7 +148,7 @@ impl Driver {
         let simulations = settlement_simulation::simulate_settlements(
             settlements
                 .iter()
-                .map(|settlement| settlement.settlement.clone().into()),
+                .map(|settlement| settlement.clone().into()),
```

Comment:
I am surprised settlement.settlement was actually doing anything before....

Reply:
The outer settlement is a "RatedSettlement { settlement: Settlement }", and now there is a coercion between the two by means of the  new "From<>" trait.